lighthouse-cli/test/fixtures/static-server.js

```diff
@@ -26,12 +26,18 @@ class Server {
     this._server = http.createServer(this._requestHandler.bind(this));
     /** @type {(data: string) => string=} */
     this._dataTransformer = undefined;
+    this._baseDir = __dirname;
   }
 
   getPort() {
     return this._server.address().port;
   }
 
+  /** @param {string} baseDir */
```

Comment:
seems this could just be a (public) class field.

Reply:
oooh yeah will do